test/qunit/model/FHIRModel.unit.js

```diff
@@ -526,6 +526,7 @@ sap.ui.define([
 		assert.equal(mRequestHandles.direct[0].getData(), JSON.stringify(oPatient2));
 		assert.equal(mRequestHandles.direct[1].getUrl(),"https://example.com/fhir/Patient/27f89dba-b3ee-465d-aec4-b732da01ead5?_format=json");
 		assert.equal(mRequestHandles.direct[1].getData(), JSON.stringify(oPatientUpdated));
+		assert.equal(mRequestHandles.direct[1].getHeaders()["If-Match"], "W/\"" + 1 + "\"", "If-Match Header for PUT request is of correct syntax");
```

Comment:
use `strictEqual` and check the string literal without concatenation  "W/\"1\""

Reply:
done